src/main/flight/pid.c

```diff
@@ -197,6 +196,10 @@ static FAST_RAM_ZERO_INIT filterApplyFnPtr dtermLowpass2ApplyFn;
 static FAST_RAM_ZERO_INIT pt1Filter_t dtermLowpass2[2];
 static FAST_RAM_ZERO_INIT filterApplyFnPtr ptermYawLowpassApplyFn;
 static FAST_RAM_ZERO_INIT pt1Filter_t ptermYawLowpass;
+static FAST_RAM_ZERO_INIT pt1Filter_t windupLpf[3][2];
```

Comment:
The amount of filters starts getting worrisome.

Reply:
IMO it's better to have explicit filters instead of magic values hidden in code ...

Reply:
What can you do, right? If you need a filter you need a filter. In any case this is from the iterm_relax PR and if at all should be addressed there.